### PR TITLE
chore: remove dead companies

### PR DESCRIPTION
**Pull request by afonsojramos**
Remove IBAN Wallet and YouClap as they both seem to be out of business.

### CODE REVIEW

**Comment by afonsojramos** [comment]
@marmelo I still believe we can move to a only 1 reviewer needed model. In case there is lack of consensus on anything, we can always revert. And I think the benefits of having a more active repo outweigh the cons of being "less reliable".

**Review by vpereira01 — APPROVED** [review]

**Review by marmelo — APPROVED** [review]

**Comment by vpereira01** [comment]
> > @marmelo I still believe we can move to a only 1 reviewer needed model. In case there is lack of consensus on anything, we can always revert. And I think the benefits of having a more active repo outweigh the cons of being "less reliable".
> 
> I would prefer to keep 2 reviews and add more people as collaborators. But I guess there are no new collaborators and you are pushing for this change for quite some time.
> 
> What do you think @vpereira01 @pedrorijo91?

I would simplify and just require one review.

I think the rule we would want is "2 approvals for pull requests from non-collaborators, 1 for collaborators" but GitHub does not support this.
I say this because in practice "2 approvals from a random pull request" it's the same as I purposing a change (I agree with it) and just one person approving.

So enforcing just one approval and having the understanding that 2 persons should approve if the pull request is from a non-collaborator.

**Comment by marmelo** [comment]
I've reduced to 1 the required number of approvals.
Lets see how it goes! :rocket:

**Pull request merged**
